contrib/get-cadical

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#
+source "$(dirname "$0")/get-script-header.sh"
+
+if [ -e cadical ]; then
+  echo 'error: file or directory "cadical" exists; please move it out of the way.' >&2
+  exit 1
+fi
+
+git clone https://github.com/arminbiere/cadical cadical
```

Comment:
For reproducability of results/bugs, we should maybe get a specific commit? At least, we should be able to print the commit used.

Reply:
Done, added specific commit.